src/palace/manager/sqlalchemy/model/licensing.py

```diff
@@ -511,7 +511,13 @@ def with_no_delivery_mechanisms(cls, _db):
     @property
     def deliverable(self):
         """This LicensePool can actually be delivered to patrons."""
-        return (self.open_access or self.licenses_owned > 0) and any(
+        return (
+            self.open_access or self.unlimited_access or self.licenses_owned > 0
+        ) and self.has_fulfillable_delivery_mechanism
+
+    @property
+    def has_fulfillable_delivery_mechanism(self) -> bool:
```

Comment:
This method can probably be reverted since it is no longer doing anything.

Reply:
Yes. I'd revert this 👍🏻 

I think these should just be removed entirely, but lets do that in a separate PR.